client-python/grakn/service/Session/util/ResponseConverter.py

```diff
@@ -0,0 +1,384 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import abc
+from grakn.service.Session.util import enums
+from grakn.service.Session.Concept import ConceptFactory
+from grakn.exception.ClientError import ClientError
+
+
+class ResponseConverter(object):
```

Comment:
We could probably find a better name than`*Converter`. The term "converter" is too generic and used way too often. It is not immediately clear to some readers/maintainers, which direction / who are is the converter converting it to? What is it converting it to? 

Or, is "converter" the most specific we could get it to? Then perhaps the methods contained in the class could be separated to create more cohesive classes with a more meaningful name. @flyingsilverfin @lolski

Reply:
Rename to ResponseReader